bsip-0062.md

```diff
@@ -0,0 +1,44 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
+  Authors:
+    Stefan Schießl stefan.schiessl@blockchainprojectsbv.com
+    Jerry Liu bitcrab@qq.com
+  Status: Draft
+  Type: Protocol
+  Created: 2019-09-14
+  Worker: TBD
+  ```
+
+# Abstract
+Closing a short position comes with a risk of getting margin called if no spare funds are available to buy the long and reduce the debt to zero. This BSIP introduces a way to put the short position directly on the market.
+
+# Rationale
+Closing a short position with a healthy CR should come with no risk of getting margin called. Currently the user has to take out collateral from the short position in order to put it on the market. Reducing the collateral increases the risk of margin call. This is especially cumbersome and risky if you want to close a position that has CR close to MCR (in that case you can only take out a small amount of collateral, sell it, reduce debt, repeat until position closed).
+
+# Specification
+Introduce a way that user can directly place orders to sell collateral in a margin position, and the received SmartCoins will be used to reduce the debt automatically.
+
+User needs to define the amount to sell and amount to receive while placing the sell orders, and the 
+price should be greater or equal to debt/collateral, to ensure the deposit position CR will not be reduced after the filling of the sell orders.
+
+When a debt position is margin called with close short orders,system delete all the close short orders and place margin call orders. 
```

Comment:
IMHO the system should allow the user to sell collateral below MSSP. The system should allow the user to sell collateral at a very high price even if it won't be filled ever. That said, it's unnecessary to cancel the order when the debt position enters margin call territory.

Reply:
Done